src/main/java/com/gregtechceu/gtceu/utils/GTUtil.java

```diff
@@ -398,19 +403,43 @@ public static FluidStack getFluidFromContainer(Object ingredient) {
     }
 
     public static boolean canSeeSunClearly(Level world, BlockPos blockPos) {
-        if (!world.canSeeSky(blockPos.above())) {
+        if (!world.canSeeSky(blockPos.above()))
             return false;
-        }
+
         Biome biome = world.getBiome(blockPos.above()).value();
-        if (world.isRaining()) {
-            if (biome.warmEnoughToRain(blockPos.above()) || biome.coldEnoughToSnow(blockPos.above())) {
+        if (world.isRaining())
+            if (biome.warmEnoughToRain(blockPos.above()) || biome.coldEnoughToSnow(blockPos.above()))
                 return false;
-            }
-        }
-        if (world.getBiome(blockPos.above()).is(BiomeTags.IS_END)) {
+
+        if (world.getBiome(blockPos.above()).is(BiomeTags.IS_END))
             return false;
-        }
+
```

Comment:
why'd you remove the brackets?? this makes it much more difficult to see where the if statement ends, especially if it's nested like this.